.github/workflows/auto-merge-enable.yml

```diff
@@ -0,0 +1,37 @@
+# This action ensures the "auto-merge" label can only by used
```

Comment:
Let's use this thread to bikeshed label name :P

My suggestion: "merge-on-green" or something along those lines, which attempts to state the condition for the merge to happen automatically.